### DISCUSSION TITLE
Pages down for weeks, all workflows error-free, but server-side error

### DISCUSSION THREAD

**Discussion by tholonia** [community/community#82935] [Actions] [answered]
### Select Topic Area

Question

### Body

I'd like to bump a recurring issues.  I have a error-free workflow, my pages-deploy.yml is fine (incl perms), and once more, I am getting `--- layout: home # Index page ---` .  The first time it was 4 days before the site appeared.  A week ago, it happened again, but the site never reappeared.  After a few days, I republished, and the same problem.  However, on both recent publishes, the first attempt failed due to server-side Ruby error (see attached log).  Requesting a new rebuild 'fixed' that, so I have a successful workflow, but still a dead site.  It seems there are issues with the server. Do I just keep resubmitting the job in the hopes it will land on a good server?   

The basic problem seems to be:
```
An error occurred while installing google-protobuf (3.25.1), and Bundler cannot
continue.

In Gemfile:
  jekyll-theme-chirpy was resolved to 6.3.1, which depends on
    jekyll-archives was resolved to 2.2.1, which depends on
      jekyll was resolved to 4.3.3, which depends on
        jekyll-sass-converter was resolved to 3.0.0, which depends on
          sass-embedded was resolved to 1.69.5, which depends on
            google-protobuf

```
I have run the rebuild a few times, and still no page.  I then wiped the repo and rebuilt from scratch, and again, got the server-side error.  But ONLY on a new build, not on the empty "trigger rebuild" (but then why would to start up 4 days later?)  Is it possible I have a ruby plugin that I should not, and rather than warning me it is just crashing?  

I have no idea how to debug this.  Any insights?

[glog.txt](https://github.com/community/community/files/13771976/glog.txt)

**Comment by chris-j-akers** [comment]
I started to get this recently and it was resolved by updating the `.github/workflows/pages-deploy.yml` file and setting the `ruby-version` value in the `jobs` section to 3.2 (from 3.0) based on this [reddit post](https://www.reddit.com/r/github/comments/18tcwm9/git_pages_setup_ruby_action_fails_on_server/)

**Reply by nairpaa** [reply, marked as answer]
Finally, it worked! Thank you.

**Comment by guiseporaki** [comment]
If it fails again, change to version 3.3. It worked for me
